std/concurrency.d

```diff
@@ -1615,7 +1608,7 @@ class Generator(T) :
         else
         {
             static assert(0,
-                    "Fiber front is rvalue and thus cannot be moved when it defines a postblit.");
+                    "Fiber front is always rvalue and thus cannot be moved if it defines a postblit.");
```

Comment:
We know here that T defines an elaborate copy constructor so it might be better to mention it by name. Better wording for this message might be:

"Fiber front cannot be moved because " ~ t.stringof ~ " defines a postblit"

Another option is to use a template guard instead, like:
```
final T moveFront()
if (!hasElaborateCopyConstructor!T)
```